app/README.md

```diff
@@ -19,6 +19,11 @@ See also: [app roadmap](https://github.com/alexellis/growlab/issues/15)
 ![How to connect the sensor over i2c](sensor-i2c.png)
 > How to connect the sensor over i2c
 
+For the DHT22 sensor, 
```

Comment:
Is any resistor required?

Reply:
No, I use this kind DHT22 AM2302. No resistor is required. 
For example:
https://www.amazon.co.uk/gp/aw/d/B07Z6JRMCW/ref=cm_cr_arp_mb_bdcrb_top?ie=UTF8